examples/hitl/rearrange_v2/session.py

```diff
@@ -23,9 +23,10 @@ def __init__(
         episode_indices: List[int],
         connection_records: Dict[int, ConnectionRecord],
     ):
-        self.success = False
+        self.finished = False
         self.episode_indices = episode_indices
         self.current_episode_index = 0
+        self.next_session_episode = 0
```

Comment:
`current_episode_index` is an index within the episode set. If there are 1K episodes, this would be a value between 0 and 999.
`next_session_episode` is an index for the episode list (`episode_indices`). If the session contains the episodes "10, 20, 30 and 40", the possible values are "0, 1, 2, 3 or 4".

I added docstrings here to disambiguate.